src/utils/stringFormater.ts

```diff
@@ -0,0 +1,10 @@
+export const snakeToCapitalization = (item: string) => {
```

Comment:
@muhammad-bilal-ali-khokhar  why we need this?

Reply:
we have some values in snakecase like `shift_incharge` , i want show it in table `Shift Incharge`